extras/anthos-vmruntime/README.md

```diff
@@ -67,6 +69,26 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
         --project ${PROJECT_ID}
     ```
 
+- Download the Disk Image locally
+    ```sh
+    gcloud storage cp gs://${BUCKET_NAME}/pos-vm.qcow2 pos-vm.qcow2
+    ```
+
+- Enable Cloud Init using virt-sysprep
+    ```sh
+    virt-sysprep -a pos-vm.qcow2 \
+        --uninstall google-compute-engine,google-compute-engine-oslogin,google-guest-agent,google-osconfig-agent \
+        --delete '/etc/cloud/cloud.cfg.d/*.cfg' \
+        --mkdir /etc/cloud/cloud.cfg.d \
+        --write /etc/cloud/cloud.cfg.d/10_anthos.cfg:'datasource_list: [ NoCloud, ConfigDrive, None ]\n'
+    ```
+    > NOTE: In many recent Linux distributions `virt-sysprep` is part of the `libguestfs-tools` package.
```

Comment:
Can we format the `NOTE` as mentioned above?

Reply:
SG